Subject: [patch] fix data types in strlen calls
From: Robert Schwebel <r.schwebel@pengutronix.de>

request.c:73: error: pointer targets in passing argument 1 of 'strlen' differ in signedness

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>

---
 request.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: totd-1.5/request.c
===================================================================
--- totd-1.5.orig/request.c
+++ totd-1.5/request.c
@@ -70,7 +70,7 @@ int request_start (Context *cont, int tc
         if (*qname == EDNS0_ELT_BITLABEL) {
 		/* We will and may not use qname as string further */
 		qname[0] = '\0';
-	} else if (len != strlen(qname) + 1) {
+	} else if (len != strlen((const char*)qname) + 1) {
 		/* We will and may not use qname as string further */
 		qname[0] = '\0';
 	}
